src/Explorer/Controls/Editor/EditorReact.tsx

```diff
@@ -46,9 +46,24 @@ export class EditorReact extends React.Component<EditorReactProps, EditorReactSt
     }, 100);
   }
 
-  public componentDidUpdate(previous: EditorReactProps) {
-    if (this.props.content !== previous.content) {
-      this.editor?.setValue(this.props.content);
+  public componentDidUpdate() {
+    if (!this.editor) {
+      // Nothing special to do if the editor doesn't exist yet
```

Comment:
This is a style preference, but I think this line is self-documenting and makes the comment here superfluous. Generally, DE adopted the attitude that a comment is a code smell unless an underlying component/logic is just hard to understand and too easy to mess up without a comment.

Reply:
Yep, you're right, happy to tidy these up.